src/components/GlMap.svelte

```diff
@@ -137,50 +155,118 @@
       ...mapViewProps,
     });
 
-    onMapMount(map);
-
-    // Also focus map on wheel (automatically focused on click)
-    const throttledWheelHandler = throttle(() => {
-      document.getElementById(id).querySelector('canvas[tabindex="0"]').focus();
-    }, 250);
-    document
-      .getElementById(id)
-      .addEventListener('wheel', throttledWheelHandler, { passive: true });
-
-    const handleMove = ({ origin }) => {
-      const isFocused =
-        document.getElementById(id)?.contains(document.activeElement) ?? false;
-      if (isFocused) {
-        dispatch('mapMove', { options: getCurrentMapView() });
-      }
-    };
-
-    map.on('move', e => {
-      if (!e?.resize) {
-        handleMove(e);
-      }
+    // Cesium stuff below
+    // ---------------------------------------------------------------------------------
+
+    Cesium.buildModuleUrl.setBaseUrl('/Cesium/');
+
+    const viewer = new Cesium.Viewer(`cesiumContainer-${id}`, {
+      animation: false,
+      baseLayerPicker: false,
+      geocoder: false,
+      homeButton: false,
+      infoBox: false,
+      sceneModePicker: false,
+      selectionIndicator: false,
+      timeline: false,
+      navigationHelpButton: false,
+      //imageryProvider: false,
+      // creditContainer: viewer.creditDisplay.container,
+      skyBox: false,
+      skyAtmosphere: false,
+      globe: false,
+      contextOptions: {
+        webgl: {
+          alpha: true,
+        },
+      },
     });
 
-    map.on('click', e => {
-      let renderedFeatures = map.queryRenderedFeatures(e.point);
-      if (!renderedFeatures.length) return;
-      if (!isPopupOpen) {
-        popup = new glLibrary.Popup()
-          .setLngLat(e.lngLat)
-          .setHTML(getPopupHtmlString(renderedFeatures))
-          .setMaxWidth(360)
-          .addTo(map);
-
-        isPopupOpen = true;
-
-        popup.on('close', () => {
-          isPopupOpen = false;
-        });
-      } else {
-        popup.remove();
-        popup = null;
-      }
+    viewer.scene.backgroundColor = Cesium.Color.TRANSPARENT;
+
+    function zoomToAltitude(zoom) {
+      const EARTH_RADIUS = 6378137;
+      const TILE_SIZE = 512;
+
+      const altitude =
+        EARTH_RADIUS * Math.PI * (1 / Math.pow(2, zoom)) * (TILE_SIZE / 256);
+
+      return altitude * 1.5;
+    }
+
+    // Sync Cesium camera on MapLibre move
+    map.on('move', () => {
+      const center = map.getCenter();
+      const zoom = map.getZoom();
+      const pitch = map.getPitch();
+      const bearing = map.getBearing();
+
+      viewer.camera.setView({
+        destination: Cesium.Cartesian3.fromDegrees(
+          center.lng,
+          center.lat,
+          zoomToAltitude(zoom)
+        ),
+        orientation: {
+          heading: Cesium.Math.toRadians(bearing),
+          pitch: Cesium.Math.toRadians(-90 - pitch),
+          roll: 0, //Cesium.Math.toRadians(0),
+        },
+      });
```

Comment:
Haven't nailed this yet trying to translate Maplibre positions (pitch and zoom) to Cesium